dvc/ui/__init__.py

```diff
@@ -270,7 +270,7 @@ def open_browser(self, file: "StrPath") -> int:
 
         path = Path(file).resolve()
         url = (
-            relpath(path) if "Microsoft" in uname().release else path.as_uri()
+            relpath(path) if "microsoft" in uname().release else path.as_uri()
```

Comment:
```suggestion
            relpath(path) if "microsoft" in uname().release.lower() else path.as_uri()
```

may as well just use `lower()` in case MS decides to change this uname value again?